docker-compose/README.md

```
@@ -1,204 +1,279 @@
+A clean and structured guide to run **Inji Verify** locally using Docker Compose, including OpenID4VP flows and local SDK setup.
 
-  - [Install on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
-  - [Docker Desktop for Windows](https://docs.docker.com/desktop/install/windows-install/)
-  - [Other platforms](https://docs.docker.com/engine/install/)
+---
 
-- **Docker Compose**
+# 🧱 Architecture Overview
 
-  `Note: Requires installation of Docker. This step can be skippped if Docker desktop(Windows) is installed as it comes along with docker compose. Please install Docker using above links before proceeding for the installation of docker compose`
+### OPENID4VP
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)